# HG changeset patch
# User Rik <rik@octave.org>
# Date 1549816875 28800
#      Sun Feb 10 08:41:15 2019 -0800
# Node ID 87d3baf1c0244a9e478916b618da27197de2a0cf
# Parent  daa281e03d9ab9b2dc546216402c1bd66d24ccfb
Harden BIST tests against changes in save_precision().

* io.tst: Use 'save_precision (17, "local")' to save/restore precision when
saving to '-text' format.

diff --git a/test/io.tst b/test/io.tst
--- a/test/io.tst
+++ b/test/io.tst
@@ -229,16 +229,17 @@
 %! STR.scalar_fld = 1;
 %! STR.matrix_fld = [1.1,2;3,4];
 %! STR.string_fld = "Octave";
 %! STR.struct_fld.x = 0;
 %! STR.struct_fld.y = 1;
 %!
 %! struct_dat = fullfile (P_tmpdir, "struct.dat");
 %! save_default_options ("-text", "local");
+%! save_precision (17, "local");
 %! save (struct_dat, "-struct", "STR");
 %! STR = load (struct_dat);
 %!
 %! assert (STR.scalar_fld == 1 && ...
 %!         STR.matrix_fld == [1.1,2;3,4] && ...
 %!         STR.string_fld == "Octave" && ...
 %!         STR.struct_fld.x == 0 && ...
 %!         STR.struct_fld.y == 1 );
